extensions/wikia/LookupContribs/LookupContribs.php

```diff
@@ -1,8 +1,71 @@
 <?php
-define ( 'LOOKUPCONTRIBS_HELP', 'This special page can be used to display user\'s contributions on multiple wikias. For displaying the list, give username below.' ) ;
-define ( 'LOOKUPCONTRIBS_PATH', '/' ) ;
-/* cache? no cache? */
-define ( 'LOOKUPCONTRIBS_NO_CACHE', 'false' ) ;
 
-require_once ( $IP . LOOKUPCONTRIBS_PATH . "extensions/wikia/LookupContribs/SpecialLookupContribs.php" ) ;
-require_once ( $IP . LOOKUPCONTRIBS_PATH . "extensions/wikia/LookupContribs/SpecialLookupContribs_hooks.php" ) ;
```

Comment:
We usually try to avoid putting any code in extension setup file. I think we should follow this guideline and move ```LookupContribsHooks``` to it's own file.